docs/docs/slot-validation-actions.mdx

```diff
@@ -5,105 +5,32 @@ title: Slot Validation Actions
 abstract: A slot validation action is a special type of custom action, designed to handle custom extraction and/or validation of slot values. This can be used to validate slots with predefined mappings or extract slots with custom mappings.
 ---
 
-Rasa SDK provides three base classes that you can inherit from in order to implement custom slot extraction and
-validation logic:
+You can use three different types of slot validation actions to write custom extraction and validation of slot values:
 
-## `Action` class
+## Regular Custom Action
 
-You can use this class to create a regular [custom action](./custom-actions.mdx). The base class will not prevent you
-from returning arbitrary events, however a slot validation action should only return `SlotSet` and `BotUttered` events.
-Any other event type will be filtered out by the default action [`action_extract_slots`](./default-actions.mdx#action_extract_slots).
-The name of the custom action must be specified in the `action` key of the relevant
-[custom slot mapping](./domain.mdx#custom-slot-mappings) in the domain.
+You can use the `Action` Rasa SDK class to create a regular [custom action](./custom-actions.mdx).
+The base class will not prevent you from returning arbitrary events, however a slot validation action should only return
+`SlotSet` and `BotUttered` events. Any other event type will be filtered out by the default action [`action_extract_slots`](./default-actions.mdx#action_extract_slots).
+The name of the custom action must be specified in the `action` key of the relevant [custom slot mapping](./domain.mdx#custom-slot-mappings)
+in the domain.
```

Comment:
```suggestion
You can use a regular custom action [custom action](./custom-actions.mdx) that returns [`slot`](https://rasa.com/docs/action-server/events#slot) events for custom slot extraction. A slot validation action should only return
`slot` and [`bot`](https://rasa.com/docs/action-server/events#bot) events. Any other event type will be filtered out by the default action [`action_extract_slots`](./default-actions.mdx#action_extract_slots).
The name of the custom action must be specified in the `action` key of the relevant [custom slot mapping](./domain.mdx#custom-slot-mappings)
in the domain.
```

No sdk-specific stuff needed here, and event types should reflect tracker event types not Rasa SDK class names.